src/tests/ftest/util/server_utils.py

```diff
@@ -633,6 +633,36 @@ def get_single_system_state(self):
                     states, data))
         return states[0]
 
+    def check_rank_state(self, rank, valid_state, max_checks=1):
+        """Check the state of single rank in DAOS system
+
+        Args:
+            rankv(int): daos rank whose state need's to be checked
```

Comment:
Only if you need to push again
```suggestion
            rank (int): daos rank whose state needs to be checked
```